<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingReverseLensOneside</class>
 <widget class="QDialog" name="SettingReverseLensOneside">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Setting ReverseLens OneSide</string>
  </property>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_swf_64.PNG</normaloff>J:/検査画面icon/flash_swf_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxWidth">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="minimum">
    <double>-0.500000000000000</double>
   </property>
   <property name="maximum">
    <double>0.500000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.160000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>13</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Strength</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Width:0-0.5   Normally 0.16</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxStrength">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxLeftSide">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Left side</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRightSide">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>76</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Right side</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
